custom/icds_reports/utils/data_accessor.py

```diff
@@ -18,7 +19,7 @@
 def _all_zeros(data):
     values = [(kpi['value'] == 0 and kpi['all'] == 0) for row in data['records'] for kpi in row]
     if all(values):
-        notify_logger.info("ICDS - All indicators in program summary equals 0")
+        create_datadog_event('ICDS 0s', 'All indicators in program summary equals 0')
```

Comment:
I don't think datadog is the right place to log this to - I would say a sentry event with log level info would be better

Reply:
Hey @snopoke,  We explicitly moved this from sentry to datadog so the GS team would have access to the data, and so we could overlay it with existing dashboard metrics) We can add it back to sentry if you think it should also be there, or remove it from datadog if you want, but wanted a to provide a bit more context on why we chose this before you take that call.

Reply:
hmm, ok. I'll add an aggregation key then so they can be grouped in the view: https://github.com/dimagi/commcare-hq/pull/24850